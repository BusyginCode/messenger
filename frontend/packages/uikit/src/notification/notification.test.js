import React from 'react'
import { shallow } from 'enzyme'
import Notification from './notification'

describe('uikit/notification', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Notification title="title" />)
    expect(wrapper).toMatchSnapshot()
  })
})
