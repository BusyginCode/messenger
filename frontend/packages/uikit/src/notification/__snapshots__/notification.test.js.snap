// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/notification should render correctly 1`] = `
<Box
  className="general default"
  component="div"
  padding="m"
>
  <Box
    align="start"
    component="div"
    flexGrow={1}
    justify="between"
  >
    <div>
      <AText
        breed="label"
        size="m"
      >
        title
      </AText>
    </div>
  </Box>
</Box>
`;
